ecology.yaml

```diff
@@ -0,0 +1,497 @@
+---
+install_repository_dependencies: true
+install_resolver_dependencies: true
+install_tool_dependencies: false
+
+tools:
+
+  - name: 'collapse_collections'
+    owner: 'nml'
+    tool_panel_section_label: 'Collection Operations'
+
+  - name: dwt_var_perfeature
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: transformation
+    owner: ethevenot
+    tool_panel_section_label: Statistics
+
+  - name: anova
+    owner: lecorguille
+    tool_panel_section_label: Statistics
+
+  - name: heatmap
+    owner: ethevenot
+    tool_panel_section_label: Statistics
+
+  - name: univariate
+    owner: ethevenot
+    tool_panel_section_label: Statistics
+
+  - name: multivariate
+    owner: ethevenot
+    tool_panel_section_label: Statistics
+
+  - name: biosigner
+    owner: ethevenot
+    tool_panel_section_label: Statistics
+
+  - name: lcmsmatching
+    owner: prog
+    tool_panel_section_label: Annotation
+
+  - name: add_input_name_as_column
```

Comment:
Is this one really needed? Also not updated since 3 years and not condafied.

Reply:
not needed!